### PR TITLE
Add ABC supporters to index.html, subfolder 'sponsors' in img, sponsor logos

### PR DESCRIPTION
**Pull request by georgedonnelly**
I would like to merge these changes to the homepage which support the fundraiser at fund.bitcoinabc.org. Thank. you!

I have it built on a public server, DM me privately for the link to see what it looks like if you don't already have a local build.

### CODE REVIEW

**Comment by georgedonnelly** [comment]
That is actually an img alt tag. I fixed the img src tag to point to the right place, committed and force pushed. Pls take another look, thanks!

**Comment by Mengerian** [comment]
Looks a lot better now!

Two minor thoughts/suggestions:
1. May look better with the "sponsor Bitcoin ABC" image below the sponsor logos.
2. It may be good to put some sort of divider above the "Supporters" title, especially since the two article titles above have a separator between them. I'm not sure the best way to do this.

Would be good if @jasonbcox could have a look to see what he thinks of this PR.

**Comment by georgedonnelly** [comment]
Thanks for the feedback and help!

> 1. May look better with the "sponsor Bitcoin ABC" image below the sponsor logos.

I wanted to leave it there to placehold the highest category of sponsor. I can move it down tho if you want, or we can change the image itself to make it more obviously a placeholder.

> 2. It may be good to put some sort of divider above the "Supporters" title, especially since the two article titles above have a separator between them. I'm not sure the best way to do this.

I put the section in a colored boxed that uses shades of F59332.

Also included is an additional logo.

I force pushed again. Getting lots of 500 errors, not sure if it is coming through or not.

**Comment by georgedonnelly** [comment]
Thanks for the feedback and help!

> 1. May look better with the "sponsor Bitcoin ABC" image below the sponsor logos.

I wanted to leave it there to placehold the highest category of sponsor. I can move it down tho if you want, or we can change the image itself to make it more obviously a placeholder.

> 2. It may be good to put some sort of divider above the "Supporters" title, especially since the two article titles above have a separator between them. I'm not sure the best way to do this.

I put the section in a colored boxed that uses shades of F59332.

Also included is an additional logo.

I force pushed again. Getting lots of 500 errors, not sure if it is coming through or not.

File: index.html [hunk 1]
```diff
@@ -90,4 +90,13 @@ <h3 class="post-subtitle">
      </article>
     {% endfor %}
   </div>
+
+  <!-- 2020 fundraiser sponsor logos -->
+  <div id="fundraiser">
+	  <h2 style="font-size:42px;">Supporters</h2>
+	  <p>Bitcoin ABC relies on sponsorships to fund all its work, including devops, maintenance, backports, development, release management, technical support and more. Become a supporter today at <a target=_fund" href="https://fund.bitcoinabc.org/">fund.bitcoinabc.org</a>.</p>
+	  <p><a target="_sponsor1" href="https://fund.bitcoinabc.org/"><img src="img/sponsors/sponsor.png" height="100" width="500" alt="Sponsor Bitcoin ABC today at fund.bitcoinabc.org" /></a></p>
```
Comment (jasonbcox) [inline index.html:98]: Since it's the same link as above, the target should probably also be `"_fund"`

File: index.html [hunk 2]
```diff
@@ -90,4 +90,13 @@ <h3 class="post-subtitle">
      </article>
     {% endfor %}
   </div>
+
+  <!-- 2020 fundraiser sponsor logos -->
```
Comment (jasonbcox) [inline index.html:94]: You can use `<hr>` here for a horizontal divider.

File: index.html [hunk 3]
```diff
@@ -90,4 +90,13 @@ <h3 class="post-subtitle">
...
+  <div id="fundraiser">
+	  <h2 style="font-size:42px;">Supporters</h2>
+	  <p>Bitcoin ABC relies on sponsorships to fund all its work, including devops, maintenance, backports, development, release management, technical support and more. Become a supporter today at <a target=_fund" href="https://fund.bitcoinabc.org/">fund.bitcoinabc.org</a>.</p>
+	  <p><a target="_sponsor1" href="https://fund.bitcoinabc.org/"><img src="img/sponsors/sponsor.png" height="100" width="500" alt="Sponsor Bitcoin ABC today at fund.bitcoinabc.org" /></a></p>
+	  <p><a target ="_sponsor2" href="https://spicetoken.org/"><img src="img/sponsors/spice.png" height="200" width="175" alt="Visit spicetoken.org" /></a></p>
```
Comment (jasonbcox) [inline index.html:99]: The layout of the sponsors is a little wonky.  I highly recommend using flexbox, as it's industry standard now: https://css-tricks.com/snippets/css/a-guide-to-flexbox/

See my patch, which gives you a good starting point using flexbox.

File: index.html [hunk 4]
```diff
@@ -90,4 +90,13 @@ <h3 class="post-subtitle">
      </article>
     {% endfor %}
   </div>
+
+  <!-- 2020 fundraiser sponsor logos -->
+  <div id="fundraiser">
+	  <h2 style="font-size:42px;">Supporters</h2>
+	  <p>Bitcoin ABC relies on sponsorships to fund all its work, including devops, maintenance, backports, development, release management, technical support and more. Become a supporter today at <a target=_fund" href="https://fund.bitcoinabc.org/">fund.bitcoinabc.org</a>.</p>
```
Comment (jasonbcox) [inline index.html:97]: target is missing quotes

File: index.html (hunk 2 above)
Comment (georgedonnelly) [reply index.html:94]: Not a fan of hr but ok.

File: index.html (hunk 1 above)
Comment (georgedonnelly) [reply index.html]: The patch did not apply against index.html so applied it manually. Thanks for your help.

File: index.html (hunk 4 above)
Comment (georgedonnelly) [reply index.html]: checked all targets, seem to be good now.

**Comment by georgedonnelly** [comment]
I *think* it is good now, thanks for your help and feedback.

**Comment by Mengerian** [comment]
LGTM

**Pull request merged**